README.md

```diff
@@ -1,189 +1,106 @@
-
-The project directories are as follows:
- * `base`: this directory contains the abstract classes to build a model
- and a trainer. Any useful abstract class should go in this folder.
- * `models`: classes implementing the TF models (e.g. Fully-Convolutional-Network,
- GANs, seq2seq, ...). These classes inherit and implement the `BaseModel` abstract
- class.
- * `trainers`: classes handling the training actions for each TF model.
- The same trainer class could be used to train different models. These
- classes inherit the `BaseTrainer` abstract class. An out-of-bag
- cross-validation evaluation is implemented, to allow evaluation of the
- models on held-out data at each training epoch. This allows to estimate
- model over-fitting during training. Currently the held-out data is the
- same for all training epochs and it's sampled without replacement.
- * `predictors`: classes to handle predictions once the model has been
- trained. The predictors accept a frozen `.pb` file and perform the
- prediction of the specified items of an `EOPatch`, and save result in
- an EOPatch.
- * `data_loader`: these classes handle the loading of the input data into
- batches. These classes are specific to the problem and data at hand.
- * `utils`: collection of utility function such as configuration file
- parser, directory creation, and logging functionality to TensorBoard.
- * `mains`: this folder holds the scripts that allow to run specific
- projects and experiments, by chaining `models`, `data_loaders` and
- `trainers`.
- * `configs`: folder storing the configuration files completely
- determining the behaviour of a project/experiment. Config parameters are
- stored in .json files. Results of an experiment should be reproducible
- by re-running the same config file.
-
-For more details on the structure of the projects and on how to create
-new models see [Tensorflow-Project-Template](https://github.com/MrGemy95/Tensorflow-Project-Template).
-
-## Template and example files
-
-Template files `trainers/template_trainer.py` and
-`models/template_model.py` are available as guidelines on how the methods for
-**trainers** and **models** should be implemented.
-
-A toy example is available in `mains/fcn_main.py`. It uses the
-`configs/example.json` configuration file, the `data_loader/data_generator.py`,
-`models/example_model.py` and `trainers/example_trainer.py`.
+This repository provides a templated structure to generate TensorFlow projects. Using the same base structure for all TF projects should benefit model creation, debugging and experiments reproducibility.
+
+The project contains the package `eoflow` which contains the base abstract classes and implements the common models, tasks and input methods. Custom models tasks and input methods can also be implemented building on top of the provided abstract classes.
+
+The package can be installed by running the following command in the root directory of the project.
+```
+$ pip install eoflow
```

Comment:
Yeah, you're right, `pip install .` is the correct way from the local repository.

I will add the second option as the default. Thanks, I didn't know it can be done this way.